fxa-oauth-server/lib/metrics/amplitude.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This module contains mappings from activity/flow event names to
```

Comment:
The "activity/flow event" part of this comment seems inaccurate for the usage here. The strings you're transforming from are bespoke for the Amplitude events, which highlights the point that much of the abstraction is redundant for your usage. We can stick with it for now, but it seems a shame that you're jumping through unnecessary extra hoops that were specifically put in place for other services.

Reply:
Updated